libdecnumber/configure.ac

```diff
@@ -0,0 +1,109 @@
+# configure.ac for libdecnumber			-*- Autoconf -*-
+# Process this file with autoconf to generate a configuration script.
+
+# Copyright (C) 2005-2016 Free Software Foundation, Inc.
+
+# This file is part of GCC.
+
+# GCC is free software; you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free
+# Software Foundation; either version 3, or (at your option) any
+# later #version.
+
+# GCC is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License #for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with GCC; see the file COPYING3.  If not see
+# <http://www.gnu.org/licenses/>.
+
+AC_PREREQ(2.64)
+AC_INIT(libdecnumber, [ ], gcc-bugs@gcc.gnu.org, libdecnumber)
```

Comment:
Should we modify the contact parameter?
I'm not sure if it's a good idea to refer to gcc-bugs...